### PR TITLE
Fix LandmarkType in InfoBar

### PR DESCRIPTION
**Pull request by bkudiess**
Fixes Landmark type in InfoBar:
- Fix SetLocalizedLandmarkType never being called inside if
- Sets landmark type to custom 
- Moves landmark from title to grid

Before:
image

After:
image

Fixes internal bug 38393708.

### CODE REVIEW

**Review by chingucoding — APPROVED** [review]
LGTM, though I'm curios if we still need the border element if the internal grid is handling most of the things now.

File: dev/InfoBar/InfoBar.xaml [hunk 1]
```diff
@@ -106,12 +106,13 @@
                         </VisualStateManager.VisualStateGroups>
 
                         <!-- Background is used here so that it overrides the severity status color if set. -->
-                        <Grid
+                        <Grid x:Name="ContentRoot"
```
Comment (ranjeshj) [inline dev/InfoBar/InfoBar.xaml:109]: Why the change to move the name ?

**Review by ranjeshj — APPROVED** [review]
